docs/src/SUMMARY.md

```diff
@@ -4,6 +4,19 @@
 
 - [Introduction](./README.md)
 - [Getting Started](./manual.md)
+    1. [Shall I use Apalache or TLC?](./manual.md#apalacheOrTlc)
```

Comment:
Unfortunately this won't work. We'll need each subchapter to be in its own file.

Reply:
I see. Let's split the manual into several chapters then.

Reply:
I'll do this in a separate PR, since it's an orthogonal change, and get the change on top of #409